engine/Shopware/Components/Translation.php

```diff
@@ -391,6 +396,295 @@ public function write($language, $type, $key = 1, $data = null, $merge = false)
     }
 
     /**
+     * Helper method to translate orders.
+     * Method extracts all documents, payment methods and dispatch methods of the order
+     * and translates them.
+     *
+     * @param $orders
+     * @param $context
+     * @param $language integer Language id
+     * @param $fallback integer Id of fallback language
```

Comment:
The PHPdoc `@param`s should start with their typehint (e.g. `int`), followed by the parameter name and an optional description